uppsrc/CtrlCore/X11Wnd.cpp

```diff
@@ -55,7 +55,8 @@ static int s_starttime;
 void Ctrl::DoPaint(const Vector<Rect>& invalid)
 {
 	GuiLock __;
-	if(IsVisible()) {
+	Top *top = GetTop();  //aris002
+	if(top && IsVisible()) {
```

Comment:
The same do not use top. Use top and utop variables from Ctrl class.

Reply:
Same as https://github.com/ultimatepp/ultimatepp/pull/126#discussion_r1059701436